src/scripts/main.js

```diff
@@ -3,7 +3,46 @@
 const calendar = document.querySelector('#calendar');
 
 function calendarTable(year, month, element) {
-  // WRITE YOUR CODE HERE
+  const daysInMonth = new Date(year, month, 0).getDate();
+  const firstDay = new Date(year, month - 1, 1).getDay();
+  const lastDay = new Date(year, month - 1, daysInMonth).getDay();
+
+  const table = document.createElement('table');
+
+  // Header row
+  const headerRow = document.createElement('tr');
+  const days = ['пн', 'вт', 'ср', 'чт', 'пт', 'cб', 'нд'];
+
+  for (let i = 0; i < days.length; i++) {
+    const headerItem = document.createElement('th');
+
+    headerItem.innerHTML = days[i];
+    headerRow.append(headerItem);
+  }
+
+  table.append(headerRow);
+
+  // Rows of weeks
+  const numberOfWeeks = daysInMonth === 28 && firstDay === 1 ? 4 : 5;
```

Comment:
```suggestion
  const numberOfWeeks = Math.ceil((daysInMonth + firstDay) / 7)
```

Reply:
Because, you could have the start of the month on Sunday and then you should render 6 rows